Darwin/StitchCore/StitchCore/Core/Auth/Internal/StitchUserImpl.swift

```diff
@@ -22,20 +21,24 @@ internal final class StitchUserImpl: CoreStitchUserImpl, StitchUser {
          withProviderType providerType: StitchProviderType,
          withProviderName providerName: String,
          withUserProfile userProfile: StitchUserProfile,
-         withAuth auth: StitchAuthImpl) {
+         withAuth auth: StitchAuthImpl,
+         withIsLoggedIn isLoggedIn: Bool,
+         withLastAuthActivity lastAuthActivity: Double) {
         self.auth = auth
         super.init(id: id,
                    loggedInProviderType: providerType,
                    loggedInProviderName: providerName,
-                   profile: userProfile)
+                   profile: userProfile,
+                   isLoggedIn: isLoggedIn,
+                   lastAuthActivity: lastAuthActivity)
     }
 
     // MARK: Methods
-
+    // swiftlint:disable line_length
```

Comment:
[nit] this is not a linter rule that should be disabled, unless it's a function declaration in a protocol (related to a jazzy bug). Fix the lines that are too long